core/base/array.cpp

```diff
@@ -2,12 +2,10 @@
 //
 // SPDX-License-Identifier: BSD-3-Clause
 
-#include <ginkgo/core/base/array.hpp>
-
+#include "ginkgo/core/base/array.hpp"
```

Comment:
Is it not possible to have these main headers also in `<...>` fashion ? IMO it looks a bit weird with initially having `" ..."` and then `<...>` .

Reply:
No that is not supported by clang-format